test/quantization/core/test_quantized_op.py

```diff
@@ -4243,6 +4240,23 @@ def test_qlinear_pt2e(self):
                     y_s: {y_scale}, y_zp: {y_zp}""",
                 )
 
+    @skipIfNoONEDNN
+    def test_qlinear_linear_relu_pt2e(self):
+        post_op_to_qlinear_ref_dict = {
+            'none': torch.ops.quantized.linear,
+            'relu': torch.ops.quantized.linear_relu,
+        }
+        supported_post_ops = ['none', 'relu']
+        post_op_algorithm = ''
```

Comment:
ditto: post_op_algorithms and make it a list.
```suggestion
        post_op_algorithms = []
```

Reply:
Thank you for pointing out and changed accordingly.